gbn/gbn_conn.go

```diff
@@ -21,7 +21,7 @@ var (
 const (
 	DefaultN                = 20
 	defaultHandshakeTimeout = 100 * time.Millisecond
-	defaultResendTimeout    = 100 * time.Millisecond
+	defaultResendTimeout    = 2000 * time.Millisecond
```

Comment:
2 seconds might be a little bit long especially if we will only ever increase this....

Why do we only allow increasing? I think there is value with decreasing too and then just having a sane minimum

Reply:
Changed this to 1 second instead, and modified so that that we can update the resend timeout dynamically to any value, as long as it's above the default minimum value!